src/diffusers/loaders/textual_inversion.py

```diff
@@ -382,14 +376,17 @@ def load_textual_inversion(
             if not isinstance(pretrained_model_name_or_path, list)
             else pretrained_model_name_or_path
         )
-        tokens = len(pretrained_model_name_or_paths) * [token] if (isinstance(token, str) or token is None) else token
```

Comment:
@patrickvonplaten 

It's this line of code here really confuses me - it basically replicate the tokens to match the number of state_dicts. it is a mistake no? don't think it make sense to have multiple state dicts for the same token?

Reply:
Yeah if no slow tests fail when removing it, I think it's not necessary

Reply:
@patrickvonplaten 
yeah removed it - actually will get an error from the input check if passing a single non-`None` token with multiple state dicts. Can you take a look at the implementation to see if it's ok